README.md

```diff
@@ -98,12 +99,17 @@ then:
 brew install gmp redis graphicsmagick memcached
 sudo easy_install pip && sudo pip install virtualenv
 ```
+[Install Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
 
 ##### Ubuntu:
 ```
-sudo apt-get install build-essential git-core libgmp3-dev graphicsmagick redis-server python-virtualenv python-dev memcached
+sudo apt-get install build-essential git-core libgmp3-dev graphicsmagick redis-server python-virtualenv
+python-dev memcached docker-ce
+```
+#### Pull GoAws image for local SQS / SNS fake server
+```
+docker pull pafortin/goaws
```

Comment:
@deeptibaghel we should do this as part of the `install` script process

Reply:
Should it be step 2 ?

Reply:
It will hopefully go into one of these https://github.com/mozilla/fxa-local-dev/blob/master/_scripts/install_all.sh 

when `install_all` runs you should have `docker` available